packages/block-editor/src/components/url-input/index.js

```diff
@@ -477,8 +480,19 @@ class URLInput extends Component {
 			return renderControl( controlProps, inputProps, loading );
 		}
 
+		if ( ! __nextHasNoMarginBottom ) {
+			deprecated( 'Bottom margin styles for wp.blockEditor.URLInput', {
+				since: '6.1',
+				version: '6.4',
```

Comment:
```suggestion
				since: '6.2',
				version: '6.5',
```

I guess this will now be 6.2 because 6.1 was already released.